Dockerfile

```
@@ -0,0 +1,40 @@
+# We use the same image as the runtime base image so that Python
+# version and location matches.
+FROM us-west1-docker.pkg.dev/serverless-runtimes/google-22/runtimes/python312 AS builder
```

Hadolint (2.12.0)

[warning] Always tag the version of an image explicitly

(DL3006)